### ISSUE TITLE
config.pv_net.train.iter_train == False

### ISSUE THREAD

**Issue by dustinyzr** [iMoonLab/PVRNet#9]
Sorry to bother you!
Should this function be TRUE or FALSE?
In the paper PVRNet, in the first 10 epochs,the feature extraction model is fixed and only finetune the other part.

And if 'config.pv_net.train.iter_train == True:',I only got a result of 92.0%+,but not 93.6%
if I change this line to False, the result can be 93.1% but still not 93.6%
Did I make something wrong in using your code?

Thank you for your reply!

**Comment by Hxyou** [comment]
iter_train flag means whether to iteratively train the scoring part and the other part besides the extraction model.
In my experiments, I set it to True. 
From my observation, it empirically gives more chances to output higher performance.

**Comment by dustinyzr** [comment]
> iter_train flag means whether to iteratively train the scoring part and the other part besides the extraction model.
> In my experiments, I set it to True.
> From my observation, it empirically gives more chances to output higher performance.

Yes, but I saw in the paper the train strategy is that we fix the extraction model in the first 10 epochs, then train the whole model in the other epochs. So I am confused.

So the conclusion is that **_iteratively train_** is better than then above strategy?

So, about the result, did I make something wrong in using your code?
Maybe because our different pytorch environment or hardware environment?

**Comment by Hxyou** [comment]
I guess there is nothing wrong with your code and that's regular fluctuation. 
Run it several times and see which one is better.

**Comment by dustinyzr** [comment]
> fluctuation

OK! Thank you for your help!!!
